src/Microsoft.Azure.WebJobs.Logging/Entities/InstanceTableEntity.cs

```diff
@@ -19,6 +19,8 @@ internal class InstanceTableEntity : TableEntity
 
         internal static InstanceTableEntity New(FunctionInstanceLogItem item)
         {
+            item.Truncate();
```

Comment:
This is where the truncation happens (when we're converting it to an actual table entity)